templates/home.html

```diff
@@ -17,7 +17,7 @@
   <script src = "https://code.jquery.com/ui/1.10.4/jquery-ui.js"></script>
   <script src="/include/bootstrap.js"></script>
   <script src="/include/popper.min.js"></script>
-  <script src="/home.js"></script>
+  <script src="/statics/home.js"></script>

```

Comment:
No need to have statics. Keep this as /home.js

Reply:
Done